supply/lib/supply/client.rb

```diff
@@ -293,7 +293,7 @@ def release_listings(version)
     end
 
     def latest_version(track)
-      latest_version = tracks.select { |t| t.track == Supply::Tracks::DEFAULT }.map(&:releases).flatten.reject { |r| r.name.nil? }.max_by(&:name)
+      latest_version = tracks.select { |t| t.track == Supply::Tracks::DEFAULT }.map(&:releases).flatten.reject { |r| (r&.name).nil? }.max_by(&:name)
```

Comment:
What about line 302?

Reply:
Adding the null safety resolve my issue when running `bundle exec fastlane supply init --track internal`, and it doesn't have any issue as internal track will already have prior release.
Not sure tho if this is the first upload, it may cause issue as well, as I don't have any app without any release on any track, and I don't really have permission to add a new app on my org. Do you want me to add the null safety on line 302 as well?

Reply:
We have a development application that uses only internal, alpha, and beta parts but not production. We only use this to test our POCs. I think it might be useful to add it there for this reason. Thank you!